lib/widgets/qpi/course_widget.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class CourseCard extends StatelessWidget {
+  final Color borderColor;
+  final Color cardColor;
+  final Color gradeColor;
+  final String courseTitle;
+  final String grade;
+  final VoidCallback onPressed;
+
+  CourseCard(this.borderColor, this.cardColor, this.gradeColor,
+      this.courseTitle, this.grade, this.onPressed);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: Colors.transparent,
+      height: 64,
+      child: Expanded(
+        child: Card(
+          color: cardColor,
+          elevation: 3,
+          shape: RoundedRectangleBorder(
+              borderRadius: BorderRadius.all(Radius.circular(8))),
+          child: InkWell(
+            onTap: () {
+              print('card was pressed');
+            },
+            child: Ink(
+              // Container for Text Row
+              child: Container(
+                child: Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: [
+                      Container(
+                        width: 10,
+                        height: 64,
+                        decoration: BoxDecoration(
+                          color: borderColor,
+                          borderRadius: BorderRadius.all(
+                            Radius.circular(8),
+                          ),
+                        ),
+                      ),
+                      SizedBox(
+                        width: 22,
+                      ),
+                      Expanded(
+                          flex: 10,
+                          child: Text('$courseTitle',
+                              style: Theme.of(context).textTheme.headline5)),
+                      Expanded(
```

Comment:
not sure if using `Expanded` for the rest of the items would be optimal, maybe:
```dart
Container - card color
Column
  Text - course code
  ?? - units
Spacer
Container - fixed width
  Text - align left, letter grade
IconButton
```

i also had a problem with the bottom tag (solid background text) in the event cards and what i found out is that using `Expanded > Container > Text and Container` would make the container use the maximum width -> it takes up the entire space of the card because of the `Expanded`